### DISCUSSION TITLE
What is the lifetime of a permanent room?

### DISCUSSION THREAD

**Discussion by Insoulum** [dyc3/opentogethertube#760] [Q&A] [answered]
Hi! What is the lifetime of a permanent room?

**Comment by dyc3** [comment, marked as answer]
That depends on what you mean. Permanent rooms get loaded and unloaded just like normal rooms, the difference being that the server loads the room settings from the database if it's a permanent room.

**Reply by Insoulum** [reply]
I meant is there an inactivity timer, after which the URL of the room will become free for other people, and another person will be able to become the new owner. I'm sorry, this is not my main language

**Reply by dyc3** [reply]
If there is a permanent room, it will remain in the database forever, regardless of whether it has an owner or not. There currently isn't a way to delete permanent rooms. Permanent rooms with no owner can be claimed if you are logged in.

**Reply by Insoulum** [reply]
> If there is a permanent room, it will remain in the database forever, regardless of whether it has an owner or not. There currently isn't a way to delete permanent rooms. Permanent rooms with no owner can be claimed if you are logged in.

Thanks!
